backend/entities/ReportTypes.cs

```diff
@@ -0,0 +1,7 @@
+namespace Pims.Dal.Entities
+{
+    public enum ReportTypes
+    {
+        SPL = 0,
```

Comment:
Add some documentation for intellisense.

Reply:
Something I've been wondering, is there value in adding comments like this?

/// PropertyTypes enum, provides the valid property types that can be used.
or this:
/// TaskModel class, provides a model to represent a task.

To me these comments are redundant, and functionally contain the same information as the actual variable name and type.

So instead of writing something like this:
// ReportTypes enum, provides the valid report types that can be used.
I guess I would rather provide something like this:
// ReportTypes enum, provides the valid project report types that can be used in the ProjectReportService.
but now I've introduced a comment that is tied to the enums current use, if someone else starts using it or the ProjectReportService stops using it the comment is inaccurate and at high risk of being missed for updates.

I don't know if I actually have a point here, maybe this can be something to discuss in knowledge sharing.

Reply:
It's mixed bagged.  Generally a comment should provide a little more information than just repeat the name.  However where the value is more relevant are auto documentation tools (i.e. Swagger).  They will include the information in their output. Without it, it is rather ugly.